### PR TITLE
prepare compliance: ensure g2p test data copied into output directory

### PR DESCRIPTION
**Pull request by bwalsh**
ensure g2p test data copied into output directory

### CODE REVIEW

**Comment by kozbo** [comment]
+1 tested, looks good.

File: scripts/prepare_compliance_data.py [hunk 1]
```diff
@@ -242,16 +243,23 @@ def run(self):
 
         # add g2p featureSet
         g2pPath = os.path.join(self.inputDirectory, "cgd")
+        # copy all files input directory to output path
+        outputG2PPath = os.path.join(
+          self.outputDirectory, "cgd")
```
Comment (jeromekelleher) [inline scripts/prepare_compliance_data.py]: Is this three spaces? Surprising flake8 didn't flag this.

**Comment by jeromekelleher** [comment]
Looks good to me, +1, after formatting nitpick.

File: scripts/prepare_compliance_data.py (hunk 1 above)
Comment (bwalsh) [inline scripts/prepare_compliance_data.py]: thanks for the review.  fixed

**Comment by kozbo** [comment]
@bwalsh Update with master please

**Pull request merged**